store/tikv/client_batch.go

```diff
@@ -249,23 +252,33 @@ func (c *batchCommandsClient) batchRecvLoop(cfg config.TiKVClient) {
 	for {
 		resp, err := c.recv()
 		if err != nil {
+			b := NewBackoffer(context.Background(), math.MaxInt32)
 			now := time.Now()
-			for { // try to re-create the streaming in the loop.
+			skipLog := false
+			for count := 0; ; count++ { // try to re-create the streaming in the loop.
 				if c.isStopped() {
 					return
 				}
-				logutil.BgLogger().Error(
-					"batchRecvLoop error when receive",
-					zap.String("target", c.target),
-					zap.Error(err),
-				)
 
-				if c.reCreateStreamingClient(err) {
-					break
+				if count <= 10 {
+					logutil.BgLogger().Error(
+						"batchRecvLoop error when receive",
+						zap.String("target", c.target),
+						zap.Error(err),
+					)
+					if count == 10 {
+						logutil.BgLogger().Error("meet error for 10 times and don't show the same log any more")
+						skipLog = true
+					}
 				}
 
-				// TODO: Use a more smart backoff strategy.
-				time.Sleep(time.Second)
+				err1 := c.reCreateStreamingClient(err, skipLog)
+				if err1 == nil {
+					break
+				}
+				if err := b.Backoff(boTiKVRPC, err1); err != nil {
+					logutil.BgLogger().Error("backoff error", zap.Error(err))
```

Comment:
This line seems to be duplicate to https://github.com/pingcap/tidb/blob/c8d1ff7/store/tikv/backoff.go#L313

Reply:
Actually, the code will never run to this line because I set the max backoff time to be `math.MaxInt32`.
If I don't do it, the go lint check would complain `err` is not used.

Reply:
```
_ = b.Backoff(boTiKVRPC, err1)
```
And put a comment here for no further confusing?

Reply:
Our go lint check forbid us to skip error check!